### DISCUSSION TITLE
Re-executing a bunch only possible by closing items of a bunch

### DISCUSSION THREAD

**Discussion by MorHeene** [ttscoff/bunch-docs#24] [Q&A] [answered]
I noticed I can only re-execute a bunch that opens a website and sends some Vim-like keystrokes (using https://github.com/ueokande/vim-vixen) after I closed the website or the browser when I close the items of the associated bunch with “Close items in…”. While this behavior makes sense to me from a technical point of view, I was just wondering whether there is a more convenient way to rerun a bunch when needed, especially when one has closed a website service and wants to login again.

**Comment by ttscoff** [comment, marked as answer]
If you option click on an item in the menu, it will mark it as closed without actually closing it. Then you can select it again and reopen it. It's the same effect as "Clear Checkmarks" but affects only one Bunch instead of all of them.

It might not be applicable in the case, but if you're running the beta, you can also mark a Bunch as `toggles: false` and make it into a Bunch that only ever opens, never closes.

**Reply by MorHeene** [reply]
Sorry, a silly follow-up question: How exactly can I set `toggles: false` in the beta version? I tried to figure it out using the documentation of the recent version (https://brettterpstra.com/bunch/docs/integration/url-handler.html#urlhandlertoggle) but failed.

**Comment by ttscoff** [comment]
You need to use the docs at 
[/bunch-beta](https://brettterpstra.com/bunch-beta/). The `toggles: 
false` is a frontmatter key, you'll find notes on that under 
[/bunch-beta/docs/bunch-files/frontmatter/](https://brettterpstra.com/bunch-beta/docs/bunch-files/frontmatter/).
